SimpleWAWS/Code/CsmExtensions/CsmResourceGroupExtensions.cs

```diff
@@ -369,6 +371,13 @@ private static bool IsSimpleWaws(CsmWrapper<CsmResourceGroup> csmResourceGroup)
                 csmResourceGroup.tags != null && !csmResourceGroup.tags.ContainsKey("Bad")
                 && csmResourceGroup.tags.ContainsKey("FunctionsContainerDeployed");
         }
+        private static bool IsJenkinsResource(CsmWrapper<CsmResourceGroup> csmResourceGroup)
+        {
+            return !string.IsNullOrEmpty(csmResourceGroup.name) &&
```

Comment:
This will return true for any try resource, not necessarily jenkins, I'd either use the other generic function or harden the check here somehow